third_party/proto/tendermint/abci/types.proto

```diff
@@ -0,0 +1,407 @@
+syntax = "proto3";
+package tendermint.abci;
```

Comment:
```suggestion
package oc.tendermint.abci;
```

Reply:
This duplicated problem is not occurred  because there is not converted go file in the `third_part` case of Ostracon.

Above `duplicate proto type registered` problem is occurred in lbm-sdk and we need to fix in lbm-sdk.

Reply:
I understand. It seems that there is no problem.